sklearn/metrics/tests/test_classification.py

```diff
@@ -2946,13 +2946,13 @@ def test_balanced_accuracy_score(y_true, y_pred):
 @pytest.mark.parametrize(
     "classes", [(False, True), (0, 1), (0.0, 1.0), ("zero", "one")]
 )
-def test_classification_metric_pos_label_types(metric, classes):
+def test_classification_metric_pos_label_types(metric, classes, global_random_seed):
```

Comment:
```suggestion
def test_classification_metric_pos_label_types(metric, classes):
```
I think there is no need to apply `global_random_seed` here, since this test only tests if different types of `pos_label` work, which is not a mathematical property.